cmd/talosctl/cmd/mgmt/cluster/create/command.go

```diff
@@ -262,161 +276,317 @@ func init() {
 		docker: dockerOps{},
 		qemu:   qemuOps{},
 	}
+	legacyOps := legacyOps{}
+	bootMethod := ""
+
+	// getBasicCommonFlags returns the common flags that are also present on the user-facing commands.
+	getBasicCommonFlags := func() *pflag.FlagSet {
+		common := pflag.NewFlagSet("common", pflag.PanicOnError)
+
+		common.StringVar(&ops.common.networkCIDR, networkCIDRFlag, "10.5.0.0/24", "CIDR of the cluster network (IPv4, ULA network for IPv6 is derived in automated way)")
+		common.IntVar(&ops.common.controlPlanePort, controlPlanePortFlag, constants.DefaultControlPlanePort, "control plane port (load balancer and local API port)")
```

Comment:
nuke it, in-cluster use 6443 always, bind to a random port on the bridge/Docker host port mapping